Examples/BookingSystem.AspNetCore/Feeds/FacilitiesFeeds.cs

```diff
@@ -43,29 +45,134 @@ x.Modified < (DateTimeOffset.UtcNow - new TimeSpan(0, 0, 2)).UtcTicks)
 
                 var query = db
                     .SelectMulti<FacilityUseTable, SellerTable>(q)
-                    .Select(result => new RpdeItem<FacilityUse>
+                    .Select(result =>
                     {
-                        Kind = RpdeKind.FacilityUse,
-                        Id = result.Item1.Id,
-                        Modified = result.Item1.Modified,
-                        State = result.Item1.Deleted ? RpdeState.Deleted : RpdeState.Updated,
-                        Data = result.Item1.Deleted ? null : new FacilityUse
+                        var faker = new Faker() { Random = new Randomizer(((int)result.Item1.Modified + (int)result.Item1.Id)) };
```

Comment:
Apologies I don't have enough context to fully understand this. Two potential issues that I see with the random seed value provided here, though I likely misunderstand things are:

- modified + id. There will be clashes with items. Maybe that's okay or maybe there won't due to how modified and ID are set.
- modified + id. Due to the inclusion of the `modified` value, it looks like all of an item's data will change when modified changes

Reply:
Sorry Luke I don't understand the first point. How would modified+id clash? The ids are unique within the type (SessionSeries/FacilityUse etc) as the id is the primary key in the table they are stored in. Clashing across types is not an issue as they don't have shared fields.

Completely agree however the modified can change. I have changed the seed to just be the id which will be unique

Reply:
> I have changed the seed to just be the id which will be unique

Great, that fixes both points!